tests/OrganizationMfaValidationTest.php

```diff
@@ -148,7 +148,202 @@ public function testAdminWithoutMfaCannotBecameMember()
+        } catch (ClientException $e) {
+            $this->assertEquals('manage.mfaRequired', $e->getStringCode());
+        }
+    }
+
+    public function testLockAccessForSuperAdminIfMfaWasForced()
+    {
+        $this->client->addUserToOrganization($this->organization['id'], ['email' => $this->normalUserWithMfa['email']]);
+
+        $this->normalUserWithMfaClient->enableOrganizationMfa($this->organization['id']);
+
+        try {
+            $this->client->getOrganization($this->organization['id']);
+            $this->fail('Admin having MFA disabled should not have access to the organization');
+        } catch (ClientException $e) {
+            $this->assertEquals('manage.mfaRequired', $e->getStringCode());
+        }
+
+        try {
+            $this->client->createProject($this->organization['id'], ['name' => 'Test']);
+            $this->fail('Admin having MFA disabled should not have access to the organization');
+        } catch (ClientException $e) {
+            $this->assertEquals('manage.mfaRequired', $e->getStringCode());
+        }
+
+        try {
+            $this->client->updateOrganization($this->organization['id'], []);
+            $this->fail('Admin having MFA disabled should not have access to the organization');
+        } catch (ClientException $e) {
+            $this->assertEquals('manage.mfaRequired', $e->getStringCode());
+        }
+
+        try {
+            $this->client->listOrganizationUsers($this->organization['id']);
+            $this->fail('Admin having MFA disabled should not have access to the organization');
+        } catch (ClientException $e) {
+            $this->assertEquals('manage.mfaRequired', $e->getStringCode());
+        }
+
+        try {
+            $this->client->listOrganizationInvitations($this->organization['id']);
+            $this->fail('Admin having MFA disabled should not have access to the organization');
+        } catch (ClientException $e) {
+            $this->assertEquals('manage.mfaRequired', $e->getStringCode());
+        }
+    }
+
+    public function testSuperAdminCanDeleteOrganizationIfMfaWasForced()
+    {
+        $this->client->addUserToOrganization($this->organization['id'], ['email' => $this->normalUserWithMfa['email']]);
+
+        $this->normalUserWithMfaClient->enableOrganizationMfa($this->organization['id']);
+
+        $organization = $this->normalUserWithMfaClient->getOrganization($this->organization['id']);
+        $this->assertTrue($organization['mfaRequired']);
+
+        $organizationsListBeforeDelete = $this->client->listMaintainerOrganizations($this->testMaintainerId);
+
+        $this->client->deleteOrganization($this->organization['id']);
+
+        $organizationsListAfterDelete = $this->client->listMaintainerOrganizations($this->testMaintainerId);
+
+        $this->assertCount(count($organizationsListBeforeDelete) - 1, $organizationsListAfterDelete);
```

Comment:
V tomhle případě bych spíš udělal GET na tu organizaci a čekoval vyjímku. Vím, že jsme se o tomhle už bavili, ale tady mi to přijde jako jiný případ a ty chceš vědět, že zmizela právě ta organizace, kterou si chtěl smazat

Reply:
fixed